packages/parser/src/schemas/apigwv2.ts

```diff
@@ -13,14 +13,16 @@ const RequestContextV2Authorizer = z.object({
       accessKey: z.string().optional(),
       accountId: z.string().optional(),
       callerId: z.string().optional(),
-      principalOrgId: z.string().optional(),
+      principalOrgId: z.string().nullable(),
```

Comment:
Are there cases where this field is not there at all? Or will it always be a string or `null`?

Reply:
`nullish` is weaker and is imho preferable. 

`z.string().nullable()` is `string | null` while `z.string().nullish()` is  `string | null | undefined` 

I am not sure there are cases where we need the key to be present. Python does similar, which I misunderstood `Optional[str] = None` always set the field to `None` if it's not present.